syntax = "proto3";

package chatroom.msgpb;

message QuitGroup {
    uint32 user_id = 1;
    uint32 group_id = 2;
}